tests/testthat/test-renderer1-global-variables.R

```diff
@@ -6,20 +6,22 @@ viz <- list(scatter=ggplot()+
                          showSelected="Species",
                          data=iris))
 
+
 myScript <- 'myArray = [];
 for(var b in window) { 
   if(window.hasOwnProperty(b)) {myArray.push(b);} 
 }
-return myArray;'
+myArray;'
 
 getVariables <- function(){
-  vars <- remDr$executeScript(myScript)
+  vars <- remDr$Runtime$evaluate(myScript,returnByValue = TRUE)$result$value
```

Comment:
this pattern occurs in several places, so can you please make a helper function?
```
helper-functions.R:77:  remDr$Runtime$evaluate(sprintf("document.getElementById('%s').dispatchEvent(new CustomEvent('click'))", as.character(v)))
helper-HTML.R:90:  remDr$Runtime$evaluate(script.txt, returnByValue = TRUE)$result$value
grep: _snaps: Is a directory
test-renderer1-global-variables.R:17:  vars <- remDr$Runtime$evaluate(myScript,returnByValue = TRUE)$result$value
test-renderer1-knit-print.R:83:  a <- remDr$Runtime$evaluate(sprintf("div = document.getElementById('%s');div.getElementsByClassName('show_hide_selector_widgets')[0]", as.character(id)),returnByValue = TRUE)$result$value
test-renderer1-knit-print.R:84:  b <- remDr$Runtime$evaluate(sprintf("div = document.getElementById('%s');div.getElementsByClassName('show_hide_selector_widgets')[0]", as.character(id)),returnByValue = TRUE)$result$value
test-renderer1-knit-print.R:85:  show_hide <- remDr$Runtime$evaluate(sprintf("div = document.getElementById('%s');div.getElementsByClassName('table.legend tr.label_variable')[0]", as.character(id)),returnByValue = TRUE)$result$value
test-renderer1-knit-print.R:86:  widget <- remDr$Runtime$evaluate(sprintf("div = document.getElementById('%s');div.getElementsByClassName('table.legend tr.label_variable')[1]", as.character(id)),returnByValue = TRUE)$result$value
test-renderer1-knit-print.R:131:    remDr$Runtime$evaluate(sprintf("div = document.getElementById('%s');div.getElementsByClassName('show_hide_selector_widgets')[0].dispatchEvent(new CustomEvent('click'));", as.character("plot1top")))
test-renderer1-knit-print.R:132:    remDr$Runtime$evaluate(sprintf("div = document.getElementById('%s'); div.getElementsByClassName('selectize-input')[0].dispatchEvent(new CustomEvent('click'));", as.character("plot1top")))
test-renderer1-knit-print.R:134:    remDr$Runtime$evaluate(sprintf("div = document.getElementById('%s');div.getElementsByClassName('show_hide_selector_widgets')[0].dispatchEvent(new CustomEvent('click'));", as.character("plot1bottom")))
test-renderer1-knit-print.R:135:    remDr$Runtime$evaluate(sprintf("div = document.getElementById('%s'); div.getElementsByClassName('selectize-input')[0].dispatchEvent(new CustomEvent('click'));", as.character("plot1bottom")))
test-renderer1-url-fragment.R:35:old_address <- remDr$Runtime$evaluate("window.location.href")$result$value
test-renderer2-widerect.R:40:  remDr$Runtime$evaluate(script.txt)
test-renderer2-widerect.R:344:remDr$Runtime$evaluate("document.getElementsByClassName('show_hide_selector_widgets')[0].dispatchEvent(new CustomEvent('click'));")
test-renderer2-widerect.R:418:remDr$Runtime$evaluate("var e = document.getElementById('plot_duration_ms_year'); e.value = 0;e.dispatchEvent(new Event('change'));")
```

Reply:
@tdhock made the changes and did a commit of that